srv/components/controller/files/scripts/controller-cli.sh

```diff
@@ -443,7 +451,19 @@ parse_args()
                     shift 2
                  }
                 ;;
-
+            -n|--ntp)
+                echo "parse_args(): configure ntp" >> $logfile
+                if [ -z "$2" ]; then
+                    echo "Error: NTP server not provided" && exit 1;
+                fi
+                if [ -z "$3" ]; then
+                    echo "Error: Timezone not provided" && exit 1;
+                fi
+                ntp_server="$2"
+                ntp_tz="$3"
```

Comment:
This parameter should come from pillar data:
`system/ntp/time_zone`

Reply:
yes, this script is supposed to be called from api which is supposed to read time_server from pillar & provide it to this utility